packages/components/CONTRIBUTING.md

```diff
@@ -293,7 +293,59 @@ All components, in addition to being typed, should be using JSDoc when necessary
 
 Each component that is exported from the `@wordpress/components` package should include a `README.md` file, explaining how to use the component, showing examples, and documenting all the props.
 
-## Folder structure
+## Readme example
```

Comment:
We should add an entry in the Table of Contents at the top of the document:

```diff
diff --git a/packages/components/CONTRIBUTING.md b/packages/components/CONTRIBUTING.md
index 97bb955d8b..4bad34a1b0 100644
--- a/packages/components/CONTRIBUTING.md
+++ b/packages/components/CONTRIBUTING.md
@@ -17,6 +17,7 @@ For an example of a component that follows these requirements, take a look at [`
 - [Unit tests](#unit-tests)
 - [Storybook](#storybook)
 - [Documentation](#documentation)
+- [README example](#readme-example)
 - [Folder structure](#folder-structure)
 
 ## Compatibility
@@ -293,7 +294,7 @@ All components, in addition to being typed, should be using JSDoc when necessary
 
 Each component that is exported from the `@wordpress/components` package should include a `README.md` file, explaining how to use the component, showing examples, and documenting all the props.
 
-## Readme example
+## README example
 
 ```md
 # `ComponentName`

```